scene/debugger/scene_debugger.cpp

```diff
@@ -535,7 +535,16 @@ SceneDebuggerTree::SceneDebuggerTree(Node *p_root) {
 				}
 			}
 		}
-		nodes.push_back(RemoteNode(count, n->get_name(), n->get_class(), n->get_instance_id(), n->get_scene_file_path(), view_flags));
+
+		String class_name;
+		if (Ref<Script> script_base = n->get_script(); script_base.is_valid()) {
```

Comment:
We don't use init statement syntax in the codebase.

Reply:
Updated.